src/app/_components/Footer.tsx

```
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link, Image } from '@heroui/react';
+import { ExternalLink } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+
+const Footer = () => {
+  const t = useTranslations('Footer');
+  return (
+    <div>
+      <div className="py-8 flex flex-col items-center text-center gap-4 text-shark-900">
+        <p>Copyright © {new Date().getFullYear()}</p>
+        <p dangerouslySetInnerHTML={{ __html: t('department') }}></p>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)